OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -109,6 +132,11 @@ public Armament(Actor self, ArmamentInfo info)
 			Barrels = barrels.ToArray();
 		}
 
+		public WDist GetModifiedMaxRange()
```

Comment:
I don't think the `GetModified` prefix adds much here, so can we call this just `MaxRange()`?